packages/flutter/lib/src/animation/animation.dart

```diff
@@ -56,6 +59,15 @@ abstract class Animation<T> extends Listenable implements ValueListenable<T> {
   /// const constructors so that they can be used in const expressions.
   const Animation();
 
+  /// Create a new animation from a [ValueListenable].
+  ///
+  /// The returned animation will always have an animations status of
+  /// [AnimationStatus.forward]. The value of the provided listenable can
+  /// be optionally transformed using the [transformer] function.
```

Comment:
Bonus points if you add the example from [this comment](https://github.com/flutter/flutter/pull/108644#issuecomment-1199966400) here to showcase the API and how it can simplify things.

Reply:
Done